script/workLoggingSystem.js

```diff
@@ -125,6 +127,8 @@ $("#submitNewTask").click(function() {
     var estimatedTime = $("#estimatedTimeCreateNewTask").val();
     var status = $("#statusCreateNewTask").val();
     var category = $("#categoryCreateNewTask").val();
+    var date = new Date();
+    var loggingDate = date.getDate() + "/" + (date.getMonth()+1) + "/" + date.getFullYear();
```

Comment:
Rather than creating a new Date object the date can be fetched from the already created id an done with others.

Reply:
okay, i will do this as soon as possible.